bin/crystal

```diff
@@ -101,7 +101,7 @@ remove_path_item() {
   path="$1"
   item="$2"
 
-  echo -n $path | awk -v RS=: -v ORS=: '$0 != "'$item'"' | sed 's/:$//'
+  printf "$path" | awk -v RS=: -v ORS=: '$0 != "'$item'"' | sed 's/:$//'
```

Comment:
No but I mean actually `%s`, otherwise percent signs will be handled weirdly 
```suggestion
  printf "%s" "$path" | awk -v RS=: -v ORS=: '$0 != "'$item'"' | sed 's/:$//'
```